README.md

```diff
@@ -1,24 +1,169 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+  <br/>
+  <br/>
 
-* Ruby version
+  <h1><b>Budget App</b></h1>
 
-* System dependencies
+</div>
 
-* Configuration
+<!-- TABLE OF CONTENTS -->
 
-* Database creation
+# 📗 Table of Contents
 
-* Database initialization
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* How to run the test suite
+<!-- PROJECT DESCRIPTION -->
 
-* Services (job queues, cache servers, search engines, etc.)
+# 📖 CashTrack <a name="about-project"></a>
 
-* Deployment instructions
+Budget App is a personal finances App, that allow you to track your outgoings.
 
-* ...
+# Check the live demo [here](https://www.loom.com/share/9b139f84e922416a820fef9eff242d89)
```

Comment:
- [ ] kindly highlight a piece of code you are proud of in the video presentation. That will mean that you record another video and follow all the presentation guidelines. This requirement was requested by the previous reviewer in the review comment section.